src/mozilla_sec_eia/models/sec10k/notebooks/exhibit21_extractor.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>I think these upstream assets are hard to differentiate. Took a stab at making it a little more verbose:</p><ul><li><code>ex21_training_data</code>: Dataset of labeled Ex. 21 documents produced in Label Studio and used to train&nbsp;<code>layoutlm</code>&nbsp;. Each word in a document has an IOB format entity tag indicating one of these classes: subsidiary, location of incorporation, ownership percentage, or other.</li><li><code>ex21_validation_set</code>: Transcribed tables from Ex. 21 documents describing the expected inference output on a validation set of filings.</li><li><code>ex21_failed_parsing_metadata</code>: Metadata for any validation filings that couldn't be parsed and included in the inference dataset (usually empty)</li><li><code>ex21_inference_dataset</code>&nbsp;: Parsed filings prepped for the inference model. If running validation, this should be the validation set of filings.</li></ul><p> </p><p>If running inference on all the docs, do you still need the validation set in&nbsp;<code>ex21_validation_set</code>&nbsp;to be materialized? If you are running validation, should&nbsp;<code>ex21_inference_dataset</code>&nbsp;just contain the validation set of filings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/mozilla-sec-eia/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>